Rubberduck.Deployment/InnoSetup/Installer Build Script.iss

```diff
@@ -0,0 +1,691 @@
+#pragma include __INCLUDE__ + ";" + SourcePath + "\Includes\"
+
+#define protected
+#ifndef Config
+#define Config "Debug"
+#endif
+#define BuildDir ExtractFileDir(ExtractFileDir(SourcePath)) + "\bin\" + Config + "\"
+#define AppName "Rubberduck"
+#define AddinDLL "Rubberduck.dll"
+#define Tlb32bit "Rubberduck.x32.tlb"
+#define Tlb64bit "Rubberduck.x64.tlb"
+#define DllFullPath BuildDir + AddinDLL
+#define Tlb32bitFullPath BuildDir + Tlb32bit
+#define Tlb64bitFullPath BuildDir + Tlb64bit 
+#define AppVersion GetFileVersion(BuildDir + "Rubberduck.dll")
+#define AppPublisher "Rubberduck"
+#define AppURL "http://rubberduckvba.com"
+#define License SourcePath + "\License.rtf"
+#define OutputDirectory SourcePath + "Installers\"
+#define AddinProgId "Rubberduck.Extension"
+#define AddinCLSID "8D052AD8-BBD2-4C59-8DEC-F697CA1F8A66"
+
+; Output the defined constants to aid in verification
+#pragma message "Include: " + __INCLUDE__
+#pragma message "Config: " + Config
+#pragma message "SourcePath: " + SourcePath
+#pragma message "BuildDir: " + BuildDir
+#pragma message "AppName: " + AppName
+#pragma message "AddinDLL: " + AddinDLL
+#pragma message "DllFullPath: " + DllFullPath
+#pragma message "Tlb32bitFullPath: " + Tlb32bitFullPath
+#pragma message "Tlb64bitFullPath: " + Tlb64bitFullPath
+#pragma message "AppVersion: " + AppVersion
+#pragma message "AppPublisher: " + AppPublisher
+#pragma message "AppURL: " + AppURL
+#pragma message "License: " + License
+#pragma message "OutputDirectory: " + OutputDirectory 
+#pragma message "AddinProgId: " + AddinProgId
+#pragma message "AddinCLSID: " + AddInCLSID
+
+[Setup]
+; TODO this CLSID should match the one used by the current installer.
+AppId={{979AFF96-DD9E-4FC2-802D-9E0C36A60D09}

```

Comment:
The comment (left from the original installer script) eludes me. Can anyone else make the sense of this? What CLSID needs to be match to what? It is not currently in the `RubberduckGuids.cs`....